clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -76,41 +100,58 @@ object ElasticsearchIndex extends ModelAutoDerivation {
   val UpsertIdElasticsearchName = "upsert_id"
 
   val BookkeepingNames = Seq(
-    UpsertIdElasticsearchName,
-    EntityIdElasticsearchName
+    EntityIdElasticsearchName,
+    UpsertIdElasticsearchName
   )
 
-  def getEntityId(json: Json): String =
-    json.unsafeGet[String](EntityIdElasticsearchName)
-
   def getUpsertId(json: Json): UpsertId =
     json.unsafeGet[UpsertId](UpsertIdElasticsearchName)
 
+  implicit class IdJson(val json: Json) extends AnyVal {
```

Comment:
There's no need to use an implicit class for this; `getAsString` should only ever be used in `getId` above, so a private method would do (or even inlining the logic in the `map` call).